test/v1/connectors/ConnectorSpec.scala

```diff
@@ -134,6 +134,17 @@ trait ConnectorSpec extends UnitSpec with Status with MimeTypes with HeaderNames
         )
     }
 
+    protected def willGetWithQParams[T](url: String, parameters:Seq[(String, String)]): CallHandler[Future[T]] = {
```

Comment:
Don't need a whole new method for this. Can just add a parameter to the `willGet` method `parameters:Seq[(String, String)] = Seq()` with the default value in place so none of its usages need to be updated but we also avoid creating a whole new method

Reply:
Was experimenting and forgot to remove this!